tests/unit/tasks/test-server-test.js

```diff
@@ -45,6 +45,46 @@ describe('test server', function() {
     watcher.emit('change');
   });
 
+  it('transforms and sets defaultOptions in testem and invokes testem properly', function(done) {
```

Comment:
i would prefer if we just use promises here, rather then a mix of `done` and promises. The promise mode for mocha will nicely handle errors for us.